flumine/order/order.py

```diff
@@ -219,7 +227,7 @@ def lookup(self) -> tuple:
 
     @property
     def customer_order_ref(self) -> str:
-        return "{0}-{1}".format(self.trade.strategy.name_hash, self.id)
+        return "{0}{1}{2}".format(self.trade.strategy.name_hash, self.sep, self.id)
```

Comment:
`self.sep` doesn't exist in this class? Would result in duplicate code when we create a different order such as  betdaq/matchbook etc.

Reply:
Thanks for looking at this.

I moved sep into the base class, but chose to leave the validation inside BetfairOrder, as the validation rules are specified in the Betfair API docs and could be different for other exchanges.